ansible/Dockerfile.CentOS6

```diff
@@ -12,7 +12,9 @@ RUN echo "localhost ansible_connection=local" > /ansible/hosts
 
 RUN set -eux; \
  cd /ansible; \
- ansible-playbook -i hosts -s ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml --skip-tags="debug,hosts_file,hostname,adoptopenjdk,jenkins,nagios,superuser,docker,swap_file,crontab,adoptopenjdk_install"
+ ansible-playbook -i hosts -s ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml --skip-tags="debug,hosts_file,hostname,adoptopenjdk,jenkins,nagios,superuser,docker,swap_file,crontab,nvidia_cuda_toolkit,adoptopenjdk_install"
```

Comment:
Was the removal of nvidia_cuda_toolkit from this because you do'nt expect the dockerfiles to be run on a machine that supports it from a testing perspective?

Reply:
The main reason right now is that it bloats the docker file. The cuda toolkit installation is over 10gb. If we ever have a use case where we need the cuda kit here we can create a separate layer.

Reply:
Side note: We should probably add a comment to the top of the Dockerfile indicating that they are for codebuild and other such places, but it would depend on whether the host hardware supports it as to whether it can ebe used. Saying that, I'm ok with not having it in there for now - it certainly does bloat it!